LectureNotes/Week-04/readme.md

```diff
@@ -0,0 +1,12 @@
+# Week 4 - Password Validation and Salted Hashing
+### Application Security And Hardening
+
+[Return Home](../../../../)  
+
+[View Lecture Notes](http://coreyshuman.github.io/GeekwiseApplicationSecurity/LectureNotes/Week-04)  
+
+### Goals
+
+### Topics
```

Comment:
Topics:
- Password Salting and Hashing
- Bcrypt
- Authorization
  - Tokens
  - Cookies